src/firefly/js/visualize/saga/MouseReadoutWatch.js

```diff
@@ -16,11 +16,27 @@ import {CysConverter} from '../CsysConverter.js';
 import {mouseUpdatePromise} from '../VisMouseSync.js';
 import {getPixScaleArcSec, getScreenPixScaleArcSec, isImage, isHiPS, getFluxUnits} from '../WebPlot.js';
 import {getPlotTilePixelAngSize} from '../HiPSUtil.js';
+import {fireMouseReadoutChange} from '../VisMouseSync';
 
 
 const delay = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+const PAUSE_DELAY= 200;
 
 
+/**
+ * Readout watcher defined the algorythm to drive the mouse readout. It does the following:
+ *   - waits for a promise of a mouse event
+ *   - Has two modes lockByClick, on or off
+ *   - lockByClick off:
+ *       - on mouse move compute and fire a mouse readout object
+ *       - if mouse pauses (200 ms) and readout uses async, call the server for more data (the image pixel flux value)
+ *       - if mouse still pause, fire a updated mouse readout object
+ *   - lockByClick on:
+ *       - on mouse click compute and fire a mouse readout object
+ *       - if readout uses async, call the server for more data
+ *       - if the position has not changed, fire a updated mouse readout object
```

Comment:
Thank you.  These sort of information is very useful for people who's browsing the code.